From 9396105353342ad3a5d9e77b1e3a298c26be206f Mon Sep 17 00:00:00 2001
From: "Wood, Brian J" <brian.j.wood@intel.com>
Date: Tue, 27 Nov 2018 13:31:10 -0800
Subject: [PATCH 0870/1214] mmc: move it earlier in drivers/Makefile

This is a forward-ported patch from 4.14 PKT kernel, which
lifts the eMMC rootfs detection earlier and helps on
total kernel boot time.

Change-Id: I4a74a6fffd19a219440853df6645e1fba7456f15
Signed-off-by: Feng Tang <feng.tang@intel.com>
Signed-off-by: Wood, Brian J <brian.j.wood@intel.com>
---
 drivers/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/Makefile b/drivers/Makefile
index 30b1c2e..57f7c3e 100644
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -58,6 +58,8 @@ obj-y				+= char/
 # iommu/ comes before gpu as gpu are using iommu controllers
 obj-$(CONFIG_IOMMU_SUPPORT)	+= iommu/
 
+obj-y                          += mmc/
+
 # gpu/ comes after char for AGP vs DRM startup and after iommu
 obj-y				+= gpu/
 
@@ -129,7 +131,6 @@ obj-$(CONFIG_EISA)		+= eisa/
 obj-$(CONFIG_PM_OPP)		+= opp/
 obj-$(CONFIG_CPU_FREQ)		+= cpufreq/
 obj-$(CONFIG_CPU_IDLE)		+= cpuidle/
-obj-y				+= mmc/
 obj-$(CONFIG_MEMSTICK)		+= memstick/
 obj-$(CONFIG_NEW_LEDS)		+= leds/
 obj-$(CONFIG_INFINIBAND)	+= infiniband/
-- 
2.7.4

